src/components/Playback/Playback.tsx

```diff
@@ -67,18 +73,48 @@ export class Playback extends Component<Props, State> {
                     uiConfId: uiconfId
                 }
             });
-            this.setMedia(media);
+            if (screenMedia) {
+                this.kalturaPlayerScreen = KalturaPlayer.setup({
+                    targetId: "player-wrap_screen" + uniqueId,
+                    provider: {
+                        partnerId: partnerId,
+                        uiConfId: uiconfId
+                    }
+                });
+                this.setMedia(screenMedia, this.kalturaPlayerScreen);
+
+                this.kalturaPlayer.addEventListener("play", () => this.kalturaPlayerScreen.play());
+                this.kalturaPlayer.addEventListener("pause", () =>
+                    this.kalturaPlayerScreen.pause()
+                );
+                KalturaPlayer.getPlayers()["player-wrap_0"].addEventListener("seeking", () => {
+                    this.kalturaPlayerScreen.currentTime = this.kalturaPlayer.currentTime;
+                });
+            }
+            this.setMedia(media, this.kalturaPlayer);
         } catch (e) {
             console.error(e.message);
         }
     }
 
     render() {
+        const { screenMedia } = this.props;
+
         return (
-            <div
-                id={"player-wrap_" + uniqueId}
-                className={`xr_player-wrap ${styles["player-wrap"]}`}
-            />
+            <div className={`players-wrap ${styles["players-wrap"]}`}>
+                <div
+                    id={"player-wrap_" + uniqueId}
+                    className={`xr_player-wrap ${styles["player-wrap"]} ${
+                        screenMedia ? "player-wrap__main_controls" : ""
+                    }`}
+                />
+                {screenMedia && (
+                    <div
+                        id={"player-wrap_screen" + uniqueId}
+                        className={`xr_player-wrap player-wrap-screen ${styles["player-wrap"]}`}
+                    />
+                )}
```

Comment:
are you sure it happens in jsx? sound weird... we have it all over

Reply:
I clearly don't get 0 if the condition doesn't fit.. jsx is built for that

Reply:
when conditionally rendering a component from the JSX section, it's better to have a literal condition.
in this case, might render nothing since `screenMedia` might be `undefined`.
but, we should not get use to it, cos sometimes we might render "0" or "false".
https://www.designcise.com/web/tutorial/why-is-react-showing-0-when-conditionally-rendering-a-component

Reply:
done